### PR TITLE
I added to the CLI this commands:  plugin:list, plugin:disable & plugin:enable.

### PR DESCRIPTION
**Pull request by LucasZdv**
News available commands for the "plugin" namespace:
---
>   plugin:disable  Disable an existing plugin.
>   plugin:enable   Enable an existing plugin.
>   plugin:list     List existing plugins.

I did intensive testing to avoid any problem, however it is a simple feature, this facilitates us in debugging moments.

deepinscreenshot_20190208213238

**My English is not so good but I can give my best effort to add this to the documentation if this request is approved.**

### CODE REVIEW

File: modules/system/console/PluginList.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+            return;
+        }
+
+        // Create a new Table instance.
+        $table = new Table($this->output);
+
+        // Set the table headers.
+        $table->setHeaders([
+            'Plugin name', 'Version','Updates enabled', 'Plugin enabled'
```
Comment (LukeTowers) [inline modules/system/console/PluginList.php]: Missing a space between `'Version',` and `'Updates`

**Comment by LukeTowers** [comment]
@LucasZdv looks great! Could you submit a PR to the docs repo?

**Comment by LucasZdv** [comment]
@LukeTowers Excellent! Yes, I will tomorrow.

**Comment by w20k** [comment]
@LucasZdv waiting on your docs PR and good to go!

**Comment by LucasZdv** [comment]
Done! https://github.com/octobercms/docs/pull/356

File: modules/system/console/PluginList.php (hunk 1 above)
Comment (w20k) [reply modules/system/console/PluginList.php]: @LucasZdv could you add space: `'Version','Updates enabled'` -> `'Version', 'Updates enabled'`. 
Thanks!

**Comment by w20k** [comment]
@LukeTowers finished with testing. Docs look good.

Screenshots:
screenshot 2019-02-18 at 11 00 36
screenshot 2019-02-18 at 11 01 27
screenshot 2019-02-18 at 11 01 40

File: modules/system/console/PluginList.php (hunk 1 above)
Comment (LukeTowers) [reply modules/system/console/PluginList.php]: Needs to be fixed

**Comment by LukeTowers** [comment]
There's an extra separator at the end of the table, can that be removed? @LucasZdv

**Comment by LucasZdv** [comment]
Working on it.

File: modules/system/console/PluginList.php (hunk 1 above)
Comment (LucasZdv) [reply modules/system/console/PluginList.php]: > @LucasZdv could you add space: `'Version','Updates enabled'` -> `'Version', 'Updates enabled'`.
> Thanks!

Working on it.

**Comment by LucasZdv** [comment]
@LukeTowers https://github.com/octobercms/october/pull/4127#issuecomment-464840924 & https://github.com/octobercms/october/pull/4127#discussion_r255290029
done!

**Comment by w20k** [comment]
@LukeTowers  - confirmed ;) 

Screenshot:
screenshot 2019-02-18 at 21 09 37

File: modules/system/console/PluginList.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+        $pluginTable = [];
+
+        $row = 0;
+        foreach ($allPlugins as $plugin) {
+            $row++;
+
+            $pluginTable[] = [$plugin->code, $plugin->version, (!$plugin->is_frozen) ? 'Yes': 'No', (!$plugin->is_disabled) ? 'Yes': 'No'];
+
+            if($row < $pluginsCount) {
```
Comment (LukeTowers) [inline modules/system/console/PluginList.php]: space between if and opening parenthesis please

**Comment by LucasZdv** [comment]
@LukeTowers I await your feedback

**Comment by LucasZdv** [comment]
@LukeTowers Hi, 
the documentation is already live. but the commands are not yet available in the OctoberCMS updates.

https://octobercms.com/docs/console/commands#plugin-list-command


https://github.com/octobercms/october/tree/master/modules/system/console

**Comment by LukeTowers** [comment]
That's because the documentation is always for the latest develop branch, and the code changes aren't in master yet because Build 448 is not stable. Build 449 will be stable, so as soon as that comes out it'll be available in master (or you could just switch your site to the develop branch for now)

**Pull request merged**